### ISSUE TITLE
ModuleNotFoundError

### ISSUE THREAD

**Issue by hallur** [sdushantha/wifi-password#16]
I just installed the package with `python -m pip install --user wifi-password`. When I run `wifi-password -q -i` the following exception is thrown:
```
Traceback (most recent call last):
  File "C:\Users\Bob\AppData\Roaming\Python\Python38\site-packages\qrcode\image\pil.py", line 6, in <module>
    from PIL import Image, ImageDraw
ModuleNotFoundError: No module named 'PIL'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\Bob\AppData\Roaming\Python\Python38\Scripts\wifi-password-script.py", line 11, in <module>
    load_entry_point('wifi-password==1.0.3', 'console_scripts', 'wifi-password')()
  File "C:\Users\Bob\AppData\Roaming\Python\Python38\site-packages\wifi_password\wifi_password.py", line 108, in main
    generate_qr_code(args.ssid, password, image=args.image)
  File "C:\Users\Bob\AppData\Roaming\Python\Python38\site-packages\wifi_password\wifi_password.py", line 85, in generate_qr_code
    img = qr.make_image()
  File "C:\Users\Bob\AppData\Roaming\Python\Python38\site-packages\qrcode\main.py", line 289, in make_image
    from qrcode.image.pil import PilImage
  File "C:\Users\Bob\AppData\Roaming\Python\Python38\site-packages\qrcode\image\pil.py", line 8, in <module>
    import Image
ModuleNotFoundError: No module named 'Image'
```

**Comment by littlediobolic** [comment]
If you 'pip3 install image' does that resolve your issue? I think the library is required but not in the requirements.txt

**Comment by hallur** [comment]
Yup. That resolves the issue.

**Comment by littlediobolic** [comment]
There is a comment in another issue about it. The PIL library for some reason comes on some install of Python but not on others. Likely worth throwing it in the requirements.txt

**Comment by sdushantha** [comment]
Closing this since it has been fixed in #18
